ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -14,6 +14,13 @@ title: Release Notes for AVD 4.x.x
 
 <!-- Release notes generated using configuration in .github/release.yml at devel -->
 
+## Release 4.8.0
```

Comment:
Depending on timing of 4.8.0, ansible-core 2.17 may be out. I think it would be good to wait for that, so we bump min and max at the same time. ETA for 2.17 is May 20.

Reply:
Agreed - converting to draft and we will re-visit after May 20 once 2.17 is released. 
I will work on the PR to make it ready for 2.17.